app/models/miq_widget_set.rb

```diff
@@ -94,7 +94,7 @@ def self.sync_from_file(filename)
     lookup_attributes = {}
     lookup_attributes[:name] = attrs["name"]
     lookup_attributes[:userid] = nil
-    lookup_attributes[:group_id] = nil
+    lookup_attributes[:read_only] = true
```

Comment:
I do wish this was just plain `find_by(:name => attrs["name"], :userid => nil, ...)` - but that is out of scope / not worth waiting for.